Compatibility: Bech32m support (#1521)
bitcoinops/bitcoinops.github.io: _includes/templates/compatibility/segwit.md

  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
{% endcase %}

{:id="segwit-receive-bech32m"}
{% case segwit.receive.bech32m %}
  {% when "true" %}{:.feature-yes}

>   - **Allows receiving to bech32m segwit addresses**<br>

Reviewer: ```suggestion
  - **Allows receiving to bech32m addresses**<br>
```

>     Allows the generation of bech32m native (either P2WPKH or P2WSH) segwit receiving addresses.

Reviewer: ```suggestion
    Allows the generation of bech32m (P2TR) receiving addresses.
```

  {% when "false" %}{:.feature-no}
  - **Does not allow receiving to bech32m segwit addresses**<br>

>     Does not allow the generation of bech32 native (either P2WPKH or P2WSH) segwit receiving addresses.

Reviewer: ```suggestion
    Does not allow the generation of bech32m (P2TR) receiving addresses.
```

  {% when "na" %}{:.feature-neutral}
  - **No receiving capabilities**<br>
    There are no receiving capabilities for this service.
  {% when "untested" %}{:.feature-neutral}

>   - **Not tested: Can bech32m segwit transaction outputs be received?**<br>

Reviewer: ```suggestion
  - **Not tested: Can bech32m transaction outputs be received?**<br>
```

  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
{% endcase %}

{:id="segwit-send-bech32m"}
{% case segwit.send.bech32m %}
  {% when "true" %}{:.feature-yes}

>   - **Allows sending to bech32m P2WPKH addresses**<br>

Reviewer: ```suggestion
  - **Allows sending to bech32m addresses**<br>
```

>     Allows sending to bech32m P2WPKH native segwit addresses.

Reviewer: ```suggestion
    Allows sending to bech32m (P2TR) addresses.
```

  {% when "false" %}{:.feature-no}

>   - **Does not allow sending to bech32m P2WPKH addresses**<br>

Reviewer: ```suggestion
  - **Does not allow sending to bech32m addresses**<br>
```

>     Does not allow sending to bech32m P2WPKH native segwit addresses.

Reviewer: ```suggestion
    Does not allow sending to bech32m addresses.
```

  {% when "na" %}{:.feature-neutral}
  - **No sending capabilities**<br>
    There are no sending capabilities for this service.
  {% when "untested" %}{:.feature-neutral}

>   - **Not tested: Can transaction outputs be sent to bech32m P2WPKH addresses?**<br>

Reviewer: ```suggestion
  - **Not tested: Can transaction outputs be sent to bech32m addresses?**<br>
```
